acceptance/framework/k8s/helpers.go

```diff
@@ -139,6 +139,7 @@ func CopySecret(t *testing.T, sourceContext, destContext environment.TestContext
 		secret.ResourceVersion = ""
 		require.NoError(r, err)
 	})
+	secret.Namespace = destContext.KubectlOptions(t).Namespace
```

Comment:
This gave me an error like `the namespace of the provided object does not match the namespace sent on the request` (because I was using different namespaces in cluster 1 and cluster 2)